### DISCUSSION TITLE
Any interesting AI projects to contribute to?

### DISCUSSION THREAD

**Discussion by astandrik** [community/community#145336] [Programming Help]
Hey! I'd like to contribute to some trending open-source ai projects.
But many of that I found have tons of pull requests that nobody reviews and are never merged 😕

Would prefer typescript/node - oriented
Would you recommend anything?

**Comment by OmgRod** [comment]
Hugging Face has some repositories you may be interested in. https://github.com/huggingface

They mainly support Python but also have packages for Node.js: https://huggingface.co/docs/huggingface.js/en/index

**Comment by GabrielBianconi** [comment]
Hi @astandrik - we're building https://github.com/tensorzero/tensorzero and would love to collaborate if you're interested. We'd be happy to help brainstorm and scope tasks and can quickly review any PRs. 

> TLDR: TensorZero creates a feedback loop for optimizing LLM applications — turning production data into smarter, faster, and cheaper models.

More information about contributing [here](https://github.com/tensorzero/tensorzero/blob/main/CONTRIBUTING.md).

Most of the codebase is in Rust + Python, but there are some interesting things to build in TypeScript as well.

Please join us on Slack or Discord if you're interested and send me a message (links in repo).

**Reply by msritian** [reply]
Hi @GabrielBianconi, I'm interested in contributing to TensorZero.
Thanks!

**Reply by GabrielBianconi** [reply]
Hi @msritian - thank you for the interest, please check out:

https://github.com/tensorzero/tensorzero/blob/main/CONTRIBUTING.md

**Comment by ghostinhershell** [comment]
Thanks for posting in the GitHub Community, @astandrik!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the New to GitHub category is solely related to conversations around getting started on using GitHub and the community for beginners of GitHub, Community on Discussions, and those new to different areas of the product alike! This question should be in the Models category. I've gone ahead and moved it for you. Good luck!
